web/src/layout/map.jsx

```diff
@@ -128,10 +128,9 @@ export default () => {
     () => (zoneId) => {
       const zoneOverview = zones[zoneId][selectedTimeAggregate].overviews[selectedZoneTimeIndex];
       const zoneConfig = zones[zoneId].config;
-      if (!zoneOverview || !zoneConfig) {
-        return;
+      if (zoneOverview) {
+        dispatchApplication('co2ColorbarValue', getCO2IntensityByMode(zoneOverview, electricityMixMode));
       }
-      dispatchApplication('co2ColorbarValue', getCO2IntensityByMode(zoneOverview, electricityMixMode));
```

Comment:
Don't we still need the zoneConfig check or is that never null? :)

Reply:
We should never have a zoneOverview and no config, but since that's the case I might as well add it for readability